src/node/rpc/member_frontend.h

```diff
@@ -1110,8 +1131,21 @@ namespace ccf
           call.args);
       }
 
-      // if the vote was successful, update the proposal's state
-      proposal.state = ProposalState::ACCEPTED;
+      if (!unknown_call.has_value())
+      {
+        // if the vote was successful, update the proposal's state
+        proposal.state = ProposalState::ACCEPTED;
+      }
+      else
+      {
+        // If any function in the proposal is unknown, mark the proposal as
```

Comment:
I noticed that proposals which contained unknown function calls were still marked as `Accepted` on completion. I'm assuming that this will be fixed by the JS governance changes (proposals verified on creation?) but I've fixed it here in the meantime.